app/Http/Controllers/V2/AuditStatus/GetAuditStatusController.php

```diff
@@ -20,6 +26,92 @@ public function __invoke(Request $request, AuditableModel $auditable)
             $auditStatus->entity_name = $auditable->getAuditableNameAttribute();
         }
 
-        return AuditStatusResource::collection($auditStatuses);
+        $transformedAudits = $this->transformAudits($this->getAudits($auditable), $auditable);
+
+        $combinedData = $auditStatuses->concat($transformedAudits);
+
+        return AuditStatusResource::collection($combinedData);
+    }
+
+    private function getAudits($auditable)
+    {
+        $auditsModelInstance = $this->getModelInstance($auditable);
+        if (! $auditsModelInstance) {
+            return collect();
+        }
+
+        $audits = $auditsModelInstance->audits()
+            ->orderBy('updated_at', 'desc')
+            ->orderBy('created_at', 'desc')
+            ->get();
+
+        return $audits;
+    }
+
+    private function transformAudits($audits, $auditable)
+    {
+        return $audits->map(function ($audit) use ($auditable) {
+            $data = [
+                'id' => $audit->id,
+                'uuid' => $audit->uuid,
+                'entity_name' => $auditable->getAuditableNameAttribute(),
+                'status' => $audit->new_values['status'] ?? null,
+                'comment' => $audit->new_values['feedback'] ?? null,
+                'first_name' => $audit->user->first_name ?? null,
+                'last_name' => $audit->user->last_name ?? null,
+                'type' => 'status',
+                'is_submitted' => $audit->is_submitted ?? null,
+                'is_active' => $audit->is_active ?? null,
+                'request_removed' => $audit->request_removed ?? null,
+                'date_created' => $audit->created_at ?? null,
+                'created_by' => $audit->user_id ?? null,
```

Comment:
This kind of mapping should be handled in the resource (in this case, `AuditStatusResource`; this mapping shouldn't be needed in the controller.